lib/matplotlib/_color_data.py

```diff
@@ -1142,3 +1142,26 @@
     'whitesmoke':           '#F5F5F5',
     'yellow':               '#FFFF00',
     'yellowgreen':          '#9ACD32'}
+
+# These are the 16 colors of the Solarized palette by Ethan Schoonover.
+# See https://ethanschoonover.com/solarized/
+# License: https://github.com/altercation/solarized/blob/master/LICENSE
+# Copyright (c) 2011 Ethan Schoonover
+SOLARIZED_COLORS = {
+    'solarized-base03':              '#002b36',
```

Comment:
The namespacing method we use (cf. xkcd, tab) is "solarized:base03" (colon), not "solarized-base03" (dash).